lib/ace/ext/papeeria/spellchecker_popup.coffee

```diff
@@ -16,7 +16,7 @@ define([
   # @param {Array} corrections: array of strings with substitution options.
   # @return {Array}: array of JSONs, actually.
   convertCorrectionList = (corrections) ->
-    return ({caption: item, value: item} for item in corrections)
+    return ({caption: item, value: item, meta: item, meta_score: corrections.length - i} for item, i in corrections)
```

Comment:
thanks, replaced `meta_score` with just `score` and added TODO. I couldn't removed meta completely so I replaced it with a dummy value